bigchaindb/pipelines/block.py

```diff
@@ -0,0 +1,80 @@
+import rethinkdb as r
+from multipipes import Pipeline, Node
+
+from bigchaindb.pipelines.utils import ChangeFeed
+from bigchaindb import Bigchain
+
+
+class Block:
+
+    def __init__(self):
+        self.bigchain = Bigchain()
+        self.txs = []
+
+    def filter_tx(self, tx):
+        if tx['assignee'] == self.bigchain.me:
+            tx.pop('assignee')
+            return tx
+
+    def delete_tx(self, tx):
+        r.table('backlog')\
+         .get(tx['id'])\
+         .delete(durability='hard')\
+         .run(self.bigchain.conn)
+
+        return tx
+
+    def validate_tx(self, tx):
+        tx = self.bigchain.is_valid_transaction(tx)
```

Comment:
You asked me to make a note about this minor thing, so you can fix it as part of this PR:  The docstring for `is_valid_transaction()` says that it returns "bool: `True` if the transaction is valid, `False` otherwise" but that's wrong: it returns the transaction if the transaction is valid, and `False` otherwise.